.gitignore

```diff
@@ -1,10 +1,12 @@
 # General ignores
 .env
-demo
 node_modules
 utils/storybook/coverage
 *.log
 
 # Package specific
 packages/**/dist
 package-lock.json
+
+# Demo specific
+demo/**/*
```

Comment:
Has there been any changes to add demo folder yet?

Reply:
Not yet. TBD per PR description.